### ISSUE TITLE
A11y_Code.visualstudio.com_Docs User Guide_Accessibility_Zoom:At 200% zoom the screen content links are present at right side is not visible.

### ISSUE THREAD

**Issue by SwatiTiwari-TCS** [microsoft/vscode-docs#3731]
Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_Code.visualstudio.com_Benchmark;#A11ySev1;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#Chrome;#DesktopWeb;#MAS1.4.4;#Win10;#Visual Studio Code;#CAITestPass_May2020;

**Environment Details:**
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com
Windows Version: Windows10
Browser version: Microsoft Edge Version 84.0.516.1 (Official build) dev (64-bit)

**Repro Steps:**
1.Launch the application Url :https://code.visualstudio.com
2.Navigate to "Docs" link.
3.From left navigation, select "User Guide".
4.Navigate to "Accessibility" link.
5.Zoom the screen content to 200% and observe the issue.

**Actual:**
On zooming the screen content to 200% then all the links eg: 'VS code in action' ,'Top Extension' present at right side is not visible and' Download' link present at footer section is also not visible.

**This issue is Observe throughout the application.**

**Expected:**
On zooming the screen content to 200% then all the links eg: 'VS code in action','Top Extension' present at right side and' Download' link present at footer section should be visible as its visible at normal 100%.

**Attachment link for the reference**
https://microsoft-my.sharepoint.com/:v:/p/v-nisaho/EWPQ6q5cnB9MsmME-Etebb4B7f2Viw26FpDw-EafXM82wA?e=t6SzDR

**Comment by sbatten** [comment]
the download link did in fact leave the page, I have added it to the collapsible menu. the section links are still present on the page though they go after the article. they live in the same DOM order as 100% zoom level so I am leaving those as-is.

**Comment by ManishaC1** [comment]
Above mentioned control is not present in current environment.
Github tags:#Closed
![Screenshot (448)](https://user-images.githubusercontent.com/105198219/217456196-01a59f86-5192-49ea-991b-6cae8dc4c00d.png)

**Comment by ManishaC1** [comment]
Github tags:#Closed;
